apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -3732,27 +3125,135 @@ describe('[Users]', function () {
 			agentInfo = await getAgent(agent.user._id);
 			expect(agentInfo).to.have.property('statusLivechat', 'not-available');
 		});
+
+		describe('last owner cases', () => {
+			let room;
+
+			beforeEach(async () => {
+				await updatePermission('edit-other-user-active-status', ['admin', 'user']);
+				await updatePermission('manage-moderation-actions', ['admin', 'user']);
+			});
+
+			afterEach(async () => {
```

Comment:
here you don't need a async method, you could just return the promise from `deleteRoom` and let the testing framework await for you